packages/elements/src/chart/__test__/chart.test.js

```diff
@@ -352,6 +352,24 @@ describe('chart/Chart', function () {
         expect(labelstrokeStyle).to.equal(labels[0].strokeStyle);
       }
     });
+
+    it('Should be able to change grid line color', async function () {
+      const gridLineColor = '#FF0000';
+      el = await fixture(`<ef-chart style="--grid-line-color: '${gridLineColor}'"></ef-chart>`);
+      el.config = config.line;
+      await chartRendered(el);
+      expect(el.chart.options.scales.x.grid.color).to.equal(el.getComputedVariable('--grid-line-color'));
+      expect(el.chart.options.scales.y.grid.color).to.equal(el.getComputedVariable('--grid-line-color'));
```

Comment:
Checking the computed value would not be a test as it would be always `true`.
```suggestion
      expect(el.chart.options.scales.x.grid.color).to.equal(gridLineColor);
      expect(el.chart.options.scales.y.grid.color).to.equal(gridLineColor);
```